### DISCUSSION TITLE
Custom Idle Alert Sound Bug

### DISCUSSION THREAD

**Discussion by BuzzfeedBlue** [runelite/runelite#14910] [Q&A] [answered]
When you use a custom wav file for alert notifications (in the main runelite plugin) the first time a sound is loaded, it will play twice, and until logout, it will then only play once. If you stay logged in, play the sound (twice), and then load a new file (without logging out), it will still play twice because it has to "load for the first time" the new sound

The default system alert doesnt have this issue

**Comment by Nightfirecat** [comment]
What OS are you using, and what version?

FWIW, I experience this bug as well on Debian 11, running KDE on Xorg.

**Reply by BuzzfeedBlue** [reply]
I have the issue on both my windows 10 desktop, and windows 11 laptop both running their respective version of release 21H2

**Comment by Nightfirecat** [comment, marked as answer]
Opened #14914 to track this bug

**Comment by CalebWhiting** [comment]
The problem lies [here](https://github.com/runelite/runelite/blob/a5514641800be3ec3b5984f8e6175693a4832084/runelite-client/src/main/java/net/runelite/client/Notifier.java#L478). I don't know why we are using the Clip#loop method in the first place?

The argument to the loop method is how many times to restart the clip. On the first run the clip starts from the beginning of the clip and loops it once, making it two full plays. After the first invocation the clip starts the end of the loop, is reset once, making it play once.

I would really want to know why we're using the loop method in the first place.
